website/src/pages/3million.tsx

```
@@ -0,0 +1,72 @@
+            alignItems: 'center',
+            justifyContent: 'center',
+            position: 'fixed',
+            top: 0,
+            left: 0,
+            zIndex: 1000,
+          }}>
+          <iframe
+            src="https://my.spline.design/particles-enV3SNBPzPLQSpMKANed913N/"
+            frameborder="0"
+            width="100%"
+            height="100%"></iframe>
```

Biome (2.1.2)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Safe fix: Use a self-closing element instead.


(lint/style/useSelfClosingElements)